### PR TITLE
update /core/features page

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Updated /core/features based on the copy doc: https://docs.google.com/document/d/10S6FGVRA1R1qi9v859CnFrl8rS25Za2SjEmSpPVQkzY/edit#heading=h.lixa1zw495b2

**Don't merge yet, this is to go live on the 16th July**

## QA

- View the site in your web browser at: https://ubuntu-com-11705.demos.haus/core/features
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the changes reflect the copy doc

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5469

### CODE REVIEW

File: templates/core/features/index.html [hunk 1]
```diff
@@ -1,119 +1,68 @@
 {% extends "core/base_core.html" %}
 
 {% block title %}Features | Ubuntu Core{% endblock %}
-{% block meta_description %}Case studies and customer stories of products launched with Ubuntu Core, a secure, fully containerised Ubuntu for embedded devices that is optimised for reliable updates.{% endblock %}
+{% block meta_description %}Ubuntu Core is an operating system for embedded devices and IoT. Learn about Ubuntu core and speed up time to market for your device. Learn about the features of this OS for embedded Linux IoT.{% endblock %}
 {% block meta_copydoc %}https://docs.google.com/document/d/10S6FGVRA1R1qi9v859CnFrl8rS25Za2SjEmSpPVQkzY/edit#heading=h.2rlj58ccsw9y{% endblock meta_copydoc %}
```
Comment (minkyngkm) [inline templates/core/features/index.html:5]: ``
```suggestion
{% block meta_copydoc %}https://docs.google.com/document/d/10S6FGVRA1R1qi9v859CnFrl8rS25Za2SjEmSpPVQkzY/edit{% endblock meta_copydoc %}
```

File: templates/core/features/index.html [hunk 2]
```diff
@@ -202,9 +151,168 @@ <h2>Recovery mode</h2>
...
+            <div class="col-8">
+              <h3>Snaps</h3>
+              <p>Ubuntu Core is built from snaps, a secure, confined, dependency-free, cross-platform Linux packaging format. Snaps are entirely self-contained, even to the point of encapsulating their own file system. This means they include everything they need to run in any environment. They're used by Ubuntu Core to both compose the image that's run on a device, and to deliver consistent and reliable software updates, even to low-powered, inaccessible and remotely administered embedded and IoT systems.</p>
+              <p>
+                <a href="/core/docs/snaps-in-ubuntu-core">Learn more about snaps&nbsp;&rsaquo;</a>
+              </p>
+            </div>
+            <div class="col-4 u-vertically-center u-hide--medium u-hide--small u-align--center">
+              {{ image (
+                url="https://assets.ubuntu.com/v1/a41bcb2f-UC20_Snapcraft_2.svg",
```
Comment (minkyngkm) [inline templates/core/features/index.html:243]: The image doesn't match with the image on copy doc.
```suggestion
                url="https://assets.ubuntu.com/v1/7de21966-snapshot.svg",
```

File: templates/core/features/index.html [hunk 3]
```diff
@@ -202,9 +151,168 @@ <h2>Recovery mode</h2>
...
+        </div>
+
+        <div tabindex="0" role="tabpanel" id="snapcraft-tab" aria-labelledby="snapcraft-tab-cta" hidden="hidden">
+          <div class="row u-equal-height">
+            <div class="col-8">
+              <h3>Snapcraft</h3>
+              <p>Snapcraft is a powerful and easy to use tool for building and publishing snaps. It helps you: </p>
+              <ul class="p-list">
+                <li class="p-list__item is-ticked">Build and then publish your snaps to your IoT app store</li>
+                <li class="p-list__item is-ticked">Fine version  control of updates and releases</li>
```
Comment (minkyngkm) [inline templates/core/features/index.html:290]:
```suggestion
                <li class="p-list__item is-ticked">Fine version control of updates and releases</li>
```

**Review by minkyngkm — APPROVED** [review]
LGTM @sowasred2012 - just one image needs to be updated! Nice work! 🏅

File: templates/core/features/index.html (hunk 2 above)
Comment (sowasred2012) [reply templates/core/features/index.html]: Ah yes - updated!

**Pull request merged**